### DISCUSSION TITLE
Menu with portal in Dialog

### DISCUSSION THREAD

**Discussion by sbc640964** [tailwindlabs/headlessui#2209] [Help]
I'm using @headlessui/react:1.7.7 in chrome browser

You can see the bug here
https://codesandbox.io/s/headlessui-react-menu-example-forked-d9y765?file=/src/menu.js

The problem with Menu in a portal that exists in Dialog

I used with Menu in a portal like this https://github.com/tailwindlabs/headlessui/blob/main/packages/playground-react/pages/menu/menu-with-popper.tsx

I placed it inside a dialog

The keyboard accessibility does not work well, you will see that the keyboard moves between the last and the first item only

Apparently the problem is this section of the documentation
![image](https://user-images.githubusercontent.com/46646044/214517656-ee507bb3-1297-4276-b0db-f090c9be28a7.png)

**Comment by RobinMalfait** [comment]
Hey! Thank you for your question!
Much appreciated! 🙏

The `usePopper` file is empty and therefore the reproduction won't load. Can you fix that so that we can take a look?

That said, if you use the `Portal` component exposed by Headless UI then it seems to behave the way you want (at least in the `App.js`) file: https://codesandbox.io/s/headlessui-react-menu-example-forked-mbvsdh?file=/src/App.js

**Reply by sbc640964** [reply]
was a mistake, I forgot to save changes in the link I attached, I corrected the source files
And of course the problem is still the same as I wrote above
https://codesandbox.io/s/headlessui-react-menu-example-forked-d9y765?file=/src/App.js
